src/components/ui/Slider/Slider.tsx

```diff
@@ -0,0 +1,93 @@
+import { ChangeEvent, FC } from 'react';
+
+import { MIN_GAP, ZERO_POSITION } from '@constants';
+
+import style from './Slider.module.scss';
+
+interface SliderProps {
+  sliderMin: number;
+  sliderMax: number;
+
+  priceMin: number;
+  priceMax: number;
+  step: number;
+
+  minValue: number;
+  maxValue: number;
+
+  isDisabled: boolean;
+
+  onMinChange: (value: number) => void;
+  onMaxChange: (value: number) => void;
+}
+
+export const Slider: FC<SliderProps> = ({
+  sliderMin,
+  sliderMax,
+
+  priceMin,
+  priceMax,
+  step,
+
+  minValue,
+  maxValue,
+
+  isDisabled,
+
+  onMinChange,
+  onMaxChange,
+}) => {
+  const positionLeft =
+    sliderMin - MIN_GAP > ZERO_POSITION && !isDisabled
+      ? sliderMin - MIN_GAP
+      : ZERO_POSITION;
+
+  const positionRight =
+    sliderMax - MIN_GAP > ZERO_POSITION && !isDisabled
+      ? sliderMax - MIN_GAP
+      : ZERO_POSITION;
```

Comment:
You could create a function and pass sliderMax or sliderMin to calculate values

Reply:
Set logic to a separate function